app/src/main/res/layout/messages_toolbar.xml

```diff
@@ -21,9 +21,13 @@
     android:orientation="horizontal">
 
     <com.kickstarter.ui.views.IconButton
+      android:id="@+id/messages_toolbar_back_button"
+      style="@style/ToolbarIconBackButton" />
```

Comment:
I had to make two `IconButton`s, one for the `X` close button and one for `back arrow` button since it's very hairy to apply a `style` to a view programmatically :(